tests/bluetooth/bsim_bt/bsim_test_audio/src/common.h

```diff
@@ -32,7 +32,7 @@
 #define WAIT_SECONDS 30                         /* seconds */
 #define WAIT_TIME (WAIT_SECONDS * USEC_PER_SEC) /* microseconds*/
 
-#define WAIT_FOR(cond) while (!(cond)) { k_sleep(K_MSEC(1)); }
+#define BT_WAIT_FOR(cond) while (!(cond)) { k_sleep(K_MSEC(1)); }
```

Comment:
I would rather avoid using the `BT _` prefix on things that are not part of the public BT API, to avoid confusion. 

```suggestion
#define WAIT_FOR_VALUE(cond) while (!(cond)) { k_sleep(K_MSEC(1)); }
```

or

```suggestion
#define WAIT_FOR_COND(cond) while (!(cond)) { k_sleep(K_MSEC(1)); }
```

which is more similar to the `WAIT_FOR_FLAG` further down in this file. 

This change may however not be necessary at all depending on what the change to the `wait_for` macro ends up being.

Reply:
Done